docs/source/models/field_ionization.rst

```diff
@@ -0,0 +1,166 @@
+
+The so-called barrier-suppression ionization regime is reached for strong fields where the potential barrier binding an electron is completely suppressed.
+
+Tunneling Ionization
+--------------------
+
+Tunneling ionization describes the process during which an initially bound electron quantum-mechanically tunnels through a potential barrier of finite height.
+
+Keldysh
+"""""""
+
+.. math::
+
+    \Gamma_\mathrm{K} = \frac{\left(6 \pi\right)^{1/2}}{2^{5/4}} E_\mathrm{ip} \left( \frac{F}{(2 E_\mathrm{ip})^{3/2}} \right)^{1/2} \exp\left(-\frac{2 \left(2 E_\mathrm{ip}\right)^{3/2}}{3 F}\right)
+
+According to the equation (9) in [BauerMulser1999]_ the Keldysh ionization rate has been implemented. See also [Keldysh]_ for the original work.
+
+.. note::
+
+    Assumptions:
+
+        * low field - perturbation theory
+        * :math:`\omega_\mathrm{laser} \ll E_\mathrm{ip}`
+        * :math:`F \ll F_\mathrm{BSI}`
+        * tunneling is instantaneous
+
+
+Ammosov-Delone-Krainov (ADK)
+""""""""""""""""""""""""""""
+
+.. math::
+
+    \begin{align}
+        \Gamma_\mathrm{ADK} &= \underbrace{\sqrt{\frac{3 n^{*3} F}{\pi Z^3}}}_\text{lin. pol.} \frac{F D^2}{8 \pi Z} \exp\left(-\frac{2Z}{3n^{*3}F}\right) \\
+        D &\equiv \left( \frac{4 \exp Z^3}{F n^{*4}} \right)^{n^*} \hspace{2cm} n^* = \frac{Z}{\sqrt{2 E_\mathrm{ip}}}
+    \end{align}
+
+We implemented equation (7) from [DeloneKrainov]_ which is a *simplified result assuming s-states* (since we have no atomic structure implemented, yet).
+Leaving out the pre-factor distinguishes ``ADKCircPol`` from ``ADKLinPol``.
+
+    .. attention::
+
+        Take care that :math:`Z` denotes the **residual ion charge** and not the proton number of the nucleus!
+
+In the following comparison one can see the ``ADKLinPol`` ionization rates for the transition from Carbon II to III (meaning 1+ to 2+).
+For a reference the rate for Hydrogen as well as the barrier suppression field strengths :math:`F_\mathrm{BSI}` have been plotted.
+They mark the transition from the tunneling to the barrier suppression regime.
+
+.. figure:: field_ionization_comparison_c_ii_ionization.svg
+    :name: field_ionization_comparison_c_ii_ionization
+    :alt: Comparison of ADK ionization rates for Carbon-II and Hydrogen
+
+    Comparison of ADK ionization rates for Carbon-II and Hydrogen.
+
+When we account for orbital structure in shielding of the ion charge :math:`Z` according to [ClementiRaimondi1963]_ in ``BSIEffectiveZ`` the barrier suppression field strengths of Hydrogen and Carbon-II are very close to one another.
+One would expect much earlier ionization of Hydrogen due to lower ionization energy. The following image shows how this can be explained by the shape of the ion potential that is assumed in this model.
+
+.. figure:: field_ionization_effective_potentials.svg
+    :name: field_ionization_effective_potentials
+    :alt: Effective atomic potentials of Carbon-II and Hydrogen in homogenous electric field
```

Comment:
`homogenous` is actually an old-fashioned term for `homologous` from biology ^^;

thanks!